lib/src/dbs/options.rs

```diff
@@ -11,16 +14,18 @@ use std::sync::Arc;
 /// whether field/event/table queries should be processed (useful
 /// when importing data, where these queries might fail).
 
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug)]
 pub struct Options {
+	/// Current Node ID
+	pub id: Arc<Uuid>,
 	/// Currently selected NS
 	pub ns: Option<Arc<str>>,
 	/// Currently selected DB
 	pub db: Option<Arc<str>>,
 	/// Connection authentication data
 	pub auth: Arc<Auth>,
 	/// Approximately how large is the current call stack?
-	dive: u8,
+	pub dive: u8,
```

Comment:
I don't quite see why this needs to be public now; mind explaining?

Reply:
That's a great catch, thank you!
So turns out, it was because I was initialising Options with a custom Default impl. This was super unnecessary - I have instead added the auth as part of new. It is much better now. Thanks for that!